internal/blockchain/fabric/fabric.go

```diff
@@ -217,6 +221,10 @@ func (f *Fabric) Init(ctx context.Context, config config.Section, metrics metric
 	f.capabilities = &blockchain.Capabilities{}
 	f.callbacks.handlers = make(map[string]blockchain.Callbacks)
 
+	// Hard-coded cache values for now, as we're only likely to query a given contract a few times in a short span anyway
+	f.versionCache = ccache.New(ccache.Configure().MaxSize(100))
+	f.versionCacheTTL = time.Minute * 5
```

Comment:
Some context - Multiparty Manager calls `GetNetworkVersion` and then `AddFireflySubscription` back-to-back. It so happens that `AddFireflySubscription` also calls `GetNetworkVersion`. It felt inefficient to query the blockchain twice, so this theoretically avoids _that_ problem, but maybe it's overkill...